test/e2e/es_cluster_logging.go

```diff
@@ -164,6 +164,18 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 	if nodeCount == 0 {
 		Failf("Failed to find any nodes")
 	}
+	Logf("Found %d nodes.", len(nodes.Items))
+
+	// Filter out unhealthy nodes.
+	// Previous tests may have cause failures of some nodes. Let's skip
+	// 'Not Ready' nodes, just in case (there is no need to fail the test).
+	filterNodes(nodes, func(node api.Node) bool {
+		return isNodeReadySetAsExpected(&node, true)
```

Comment:
Yes, I did wonder about that -- but checked just to make sure and I did compile and run the test.